allensdk/api/cloud_cache/cloud_cache.py

```diff
@@ -502,10 +537,14 @@ def _download_file(self, file_attributes: CacheFileAttributes) -> bool:
                 with open(local_path, 'wb') as out_file:
                     for chunk in response['Body'].iter_chunks():
                         out_file.write(chunk)
+                pbar.update(response["ContentLength"])
```

Comment:
If my understanding is correct, response['ContentLength'] will always equal object_info['Size']. I think this line needs to be within the for loop that iterates through the chunks like this:
```
pbar.update(len(chunk))
```

in order to update the progress bar as it downloads the file.

Reply:
fixed in branch about to be merged here.